pkg/util/jsonutils/jsonutils_test.go

```diff
@@ -0,0 +1,48 @@
+package jsonutils
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestUpdateJsonString(t *testing.T) {
+	type test struct {
+		name             string
+		jsonString       string
+		jsonKeyValuePair map[string]string
+		expectedString   string
+		expectedChanged  bool
+	}
+	jsonStringByte, _ := json.Marshal(map[string]string{"test": "test"})
+
+	for _, tt := range []*test{
```

Comment:
It would be good to include some cases for error cases as well (can't parse json, key did not exist)